cli/cmd/generate_aws.go

```diff
@@ -494,6 +510,11 @@ func validateAwsProfile(val interface{}) error {
 	return validateStringWithRegex(val, fmt.Sprintf(`^%s$`, AwsProfileRegex), "invalid profile name supplied")
 }
 
+// survey.Validator for aws profile
+func validateAwsAssumeRole(val interface{}) error {
+	return validateStringWithRegex(val, fmt.Sprintf(`^%s$`, AwsAssumeRoleRegex), "invalid assume name supplied")
```

Comment:
Doesn't this lead to a regex of `^^ ... $$`?  `AwsAssumeRoleRegex` already has the anchors.